### ISSUE TITLE
strictlyNew = false error

### ISSUE THREAD

**Issue by bormat** [naugtur/insertionQuery#24]
Hi,
Setting strictlyNew to false seems to create the error "Cannot read property 'parentNode' of null(…)"
The incriminated function is topmostUntaggedParent which continue recursive call  even when they are no parentNode.
I use chrome.

```
<div class="b0 test"></div>

<script>
	insertionQ.config({
	    strictlyNew : false,
	    timeout : 0,
	});
	insertionQ('.test').summary(function(arrayOfInsertedNodes){
	    console.log(arrayOfInsertedNodes)
	});
	setTimeout(function(){
		$('<div class="b1 test"></div>').appendTo('body')
	},1000)

</script>

```
Thanks

**Comment by naugtur** [comment]
hmm... it was supposed to recognize when it reaches the top. Thanks for reporting, I'll look into it.

**Comment by naugtur** [comment]
Fixed, published v1.0.4
